less/navigation.less

```diff
@@ -1 +1,40 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+.navBar{
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    flex-direction: row;
+    font-size: 16px;
+    margin: 20px;
+    width:50%;
+
+    @media @mobile{
+        display: flex;
+        justify-content: space-around;
+    }
+}
+a{
+    text-decoration: none;
+    color: black;
+}
+
+.funBus{
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    flex-direction: row;
+    font-size: 4rem;
+    margin:20px;
+}
+
+.fun-bus-info{
+    margin: 20px;
+
+    @media @mobile{
```

Comment:
Great job setting up your media queries and escaping for your @mobile variable! Looks like you're using those same flex properties a bunch for mobile view, next time maybe try implementing a mixin to make your code more DRY